src/main/java/core/screens/GameScreen.java

```diff
@@ -21,7 +20,16 @@
     private float x;
     private float y;
 
-    public GameScreen() {}
+    private final HudManager hud;
+    private final GdxGame game;
+    private boolean displayInventory = false;
+    private InventoryWindow inventory;
+
+    public GameScreen(GdxGame game) {
+        this.hud = new HudManager(game.batch);
+        this.game = game;
+        this.inventory = new InventoryWindow();
```

Comment:
local property name is confusing, seems to imply that the GameplayController has its own inventory

Also per above, imo makes sense to move this to the HUDController

Reply:
Addressed above